code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -118,7 +118,7 @@
 		if(check_chunky_fingers())
 			balloon_alert(src, "fingers are too big!")
 			return FALSE
-	if(HAS_TRAIT(src, TRAIT_NOGUNS))
+	if(HAS_TRAIT(src, TRAIT_NOGUNS) && !istype(G, /obj/item/gun/ballistic/bow)) /// bow is not a gun
```

Comment:
no guns means no guns, not only some guns

Reply:
Ninja can't use guns

Reply:
yes and i dont want them to